core/src/main/scala/org/apache/spark/h2o/backends/internal/InternalBackendConf.scala

```diff
@@ -55,6 +58,7 @@ trait InternalBackendConf extends SharedBackendConf {
   def setNodeBasePort(port: Int) = set(PROP_NODE_PORT_BASE._1, port.toString)
   def setNodeIcedDir(dir: String) = set(PROP_NODE_ICED_DIR._1, dir)
   def setNodeNetworkMask(mask: String) = set(PROP_NODE_NETWORK_MASK._1, mask)
+  def setInternalSecureConnectionsEnabled(flag: Boolean) = set(PROP_INTERNAL_SECURE_CONNECTIONS._1, flag)
```

Comment:
to be consistent with all the methods in the current api of conf, this method should be
`setInternalSecureConnectionsEnabled()` ( no arguments) and there should be corresponding method `setInternalSecureConnectionsDisabled()`